src/main/java/org/sagebionetworks/web/client/widget/table/v2/results/cell/EnumFormCellEditor.java

```diff
@@ -73,33 +97,57 @@ public void fireEvent(GwtEvent<?> event) {
 
 	@Override
 	public int getTabIndex() {
-		return view.getTabIndex();
+		if (radioView != null) {
+			return radioView.getTabIndex();
+		} else {
+			return listView.getTabIndex();
+		}
 	}
 
 	@Override
 	public void setAccessKey(char key) {
-		view.setAccessKey(key);
+		if (radioView != null) {
+			radioView.setAccessKey(key);
+		} else {
+			listView.setAccessKey(key);
+		}
 	}
 
 	@Override
 	public void setFocus(boolean focused) {
-		view.setFocus(focused);
+		if (radioView != null) {
+			radioView.setFocus(focused);
+		} else {
+			listView.setFocus(focused);
+		}
 	}
 
 	@Override
 	public void setTabIndex(int index) {
-		view.setTabIndex(index);
+		if (radioView != null) {
+			radioView.setTabIndex(index);
+		} else {
+			listView.setTabIndex(index);
+		}
 	}
 
 	public void configure(List<String> validValues) {
-		/*
-		 * The items include all passed values
-		 */
+		view.clear();
 		this.items = new ArrayList<String>(validValues.size());
 		for (String value : validValues) {
 			this.items.add(value);
 		}
-		view.configure(this.items);
+		if (validValues.size() > MAX_RADIO_BUTTONS) {
```

Comment:
If configured with more than MAX_RADIO_BUTTONS items, then it uses a dropdown list view.  Otherwise, it uses the old radio buttons.